lib/types.ts

```diff
@@ -163,6 +163,16 @@ export interface PluginOptions {
 
   /** Whether to enable application dependencies scanning. The default is "false" */
   "app-vulns": boolean | string;
+
+  /**
+   * Whether to unpack jars and look for jars within them.
+   * If jar contains other jars (AKA fat-jar or shaded-jar), we send back only the children jars, and don't look for vulns in the parent.
+   * Currently only one level deep.
+   * Must always come with app-vulns
```

Comment:
Is this enforced somehow?

Reply:
hmm, only in the sense that you wont even get to the desired code if you don't have the app-vulns flag:
https://github.com/snyk/snyk-docker-plugin/pull/367/files#diff-ef7473bc4617398b357238645eb8280929cc7617ebf63ad15e8fb1ea8d82cf34L149
https://github.com/snyk/snyk-docker-plugin/pull/367/files#diff-ef7473bc4617398b357238645eb8280929cc7617ebf63ad15e8fb1ea8d82cf34L79

do you believe it's not enough?

also made sure it's covered in a test:
https://github.com/snyk/snyk-docker-plugin/pull/367/files#diff-b0203a8cdef34bc7279d600248ec84804e0b90001ca61141bb95f2bb4735bb51R106

Reply:
My point is having as few surprises as possible for whoever consumes this library.
If I provide the new flag, but not appVulns, and I see nothing, I'm wondering what's going on.
If I provide the new flag, but not appVulns, and I see an error, I understand I've done something wrong

Reply:
👍 gotcha

Reply:
added a commit https://github.com/snyk/snyk-docker-plugin/pull/367/commits/32be93a3500ee794c3d4ad5124ec18867f62eb82

I will squash all commits before merging. also will get @garethr 's feedback on the error msg.